packages/core/src/types.ts

```diff
@@ -301,15 +301,19 @@ export type Receiver<TEvent extends EventObject> = (
 export type InvokeCallback<
```

Comment:
while we are at it, could we move this type to `packages/core/src/actors/callback.ts`? Same for `Receiver`

Reply:
Makes sense. I've kept them exported, but might not be no longer necessary. Let me know what you want.